script.js

```diff
@@ -0,0 +1,73 @@
+function setBackgroundColor(heightScreen) {
+  if (heightScreen > 1000) {
+    $("#home-page").css("background-image", "none");
+    $("#home-page").css("background-color", "rgb(133, 57, 98)");
+  }
+}
+
+function assignDataContentToModals() {
+  var ascoderuStory = "Ascoderu founder & CEO, Nzola Swasisa, was born and raised in the Democratic Republic of the Congo (DRC) and at a young age he took electronics communications courses. " +
+                      "For 19 years Nzola worked for Mission Aviation Fellowship (MAF) in 5 different countries; DRC, Angola, Namibia, South Africa and Zambia. " +
+                      "At that time, Nzola became aware of the lack of telecommunication infrastructions in rural communities and he begun to refurbish old radio communications to help establish new communication systems. " +
+                      "In the late 90’s, Nzola started implementing a radio email system, which consists of sending emails via radio waves. "
+  var recStory = "When cellphones became popular in Africa, Nzola then decided to build an offline mobile email server. " +
+                 "In 2010, Nzola registered Ascoderu as a not for profit NGO in Canada and in the DRC with the mission to build the server. " +
+                 "Between 2010-15, Nzola and Jason Cobham together built the first prototype of the offline mobile email server, which they called REC “Remote Email Center”. " +
+                 "In 2013 and 2015 Nzola traveled to Kinshasa to test REC and he realized 2 things: REC was a successful proof-of-concept but it wasn't reliable nor user friendly. " +
+                 "Since 2015, new people joined Nzola's efforts to modify and improve REC and in January 2017 the second prototype was built: LOKOLE. ";
+
+  $("#info1").data("content", ascoderuStory);
+  $("#info2").data("content", recStory);
+}
+
+function checkDevice(height, width) {
```

Comment:
What does this function actually do? Seems to return a height and width? Could the name be made more explicit to better explain what this does?